frontend/src/components/pages/CreatePasswordPage.tsx

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import {
+  Input,
+  Button,
+  Center,
+  Stack,
+  Flex,
+  Box,
+  Text,
+  FormLabel,
+  FormControl,
+} from "@chakra-ui/react";
+import ResponsiveLogo from "../common/responsive/ResponsiveLogo";
+import ResponsivePasswordInput from "../common/responsive/ResponsivePasswordInput";
+import ResponsiveAuthContainer from "../common/responsive/ResponsiveAuthContainer";
+import ResponsiveModalWindow from "../common/responsive/ResponsiveModalWindow";
+import background from "../assets/background.png";
+import backgroundMobile from "../assets/background_mobile.png";
+
+const CreatePasswordPage = (): React.ReactElement => {
+  const [showModal, setShowModal] = React.useState(false);
+  const [password, setPassword] = React.useState("");
+  const [confirmPassword, setConfirmPassword] = React.useState("");
+  const [errorMessage, setErrorMessage] = React.useState("");
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+  const handleConfirmPasswordChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setConfirmPassword(event.target.value);
+  };
+
+  const validatePasswords = () => {
+    setErrorMessage("");
+    if (password.length < 8) {
+      setErrorMessage("Password must be at least 8 characters.");
+      return true;
+    }
+    if (confirmPassword.length < 8) {
+      setErrorMessage("Password must be at least 8 characters.");
+      return true;
+    }
+    if (confirmPassword && password !== confirmPassword) {
+      setErrorMessage(
+        "Your new password cannot be your previous password. Please try again.",
```

Comment:
Shouldn't this check the previous password from firebase? And did we want this feature? Seems like extra work lool. 

Also Firebase's default behaviour is not to check password history.